install.sh

```diff
@@ -223,6 +223,28 @@ EOF
   mkdir -p $VAR_DIR/storage
   mkdir -p $VAR_DIR/scripts
 
+  cat << EOF > $VAR_DIR/manifests/registries.yaml
+mirrors:
+  docker.io:
+    endpoint:
+      - http://k3d-tensorleap-registry:5000
+  gcr.io:
+    endpoint:
+      - http://k3d-tensorleap-registry:5000
```

Comment:
So all the images stored at `gcr.io` will be pulled first from the local reg and cached auto in the local reg?

Reply:
No auto caching
If they are in the local registry they will be pulled from there
docker.io will be used as a fallback (which means `engine` will never get there in the current implementation)